.circleci/config.yml

```diff
@@ -154,6 +156,20 @@ jobs:
       - run: yarn install --cwd=dashboard --frozen-lockfile
       - run: yarn --cwd=dashboard run lint
       - run: yarn --cwd=dashboard run test --maxWorkers=4 --coverage
+  test_chart:
+    docker:
+      - image: circleci/golang:1.9
```

Comment:
why golang? Isn't there anything more barebone than this?

Reply:
@migmartri yeah I was looking for one and couldn't find it: https://circleci.com/docs/2.0/circleci-images/